### ISSUE TITLE
[Accessibility: Visual formatting] The power icon is the only reference to attach an PCI device to a guest

### ISSUE THREAD

**Issue by alinefm** [kimchi-project/kimchi#864]
The power icon should be followed by a text explaining what it is for.

![372ce784e4a5aa2a](https://cloud.githubusercontent.com/assets/4921718/13953844/c402fd10-f01c-11e5-9771-46681fdc7b15.jpg)

**Comment by alinefm** [comment]
The same behavior is identified to detach the device.

![80b267fc-f01d-11e5-8beb-fcf07322f0f5](https://cloud.githubusercontent.com/assets/4921718/13955013/da2dbf6c-f021-11e5-84fc-cc5d9446a1d6.png)
 
Also, a green power icon is added to represent the device is active in the given guest.
